docs/products/postgresql/reference/pg-long-running-queries.rst

```diff
@@ -0,0 +1,78 @@
+PostgreSQL® long-running queries
+===============================
+
+Aiven does not terminate any customer queries even if they run indefinitely.  Long-running queries often cause issues by locking resources and preventing database maintenance tasks.
+
+To identify and terminate such long-running queries, you can do it from either:
+
+* `Aiven console <http://console.aiven.io>`_ / web page
+* `PostgreSQL shell <https://www.postgresql.org/docs/current/app-psql.html>`_ (``psql``)
+
+
+Aiven console
+-------------
+In the Aiven web console, you can go to the **Current Queries** tab for your service.
+
+.. image:: /images/products/postgresql/pg-long-running-queries.png
+    :alt: PostgreSQL® service overview tab in Aiven's console
+
+
+On this page, you can click **Terminate** to stop any queries directly.
+
+
+PostgreSQL® shell (psql)
+-----------------------
+You can login to your PostgreSQL® service via ``psql <service_uri>``.  Once connected, you can call the following function on the ``psql`` shell:
+
+.. code:: bash
+
+    => SELECT pg_terminate_backend(pid);
+
+
+You can use the following query to monitor currently running queries:
+
+.. code:: bash
+
+    => SELECT * FROM pg_stat_activity WHERE state <> 'idle';
+
+
+Client applications can use the `statement_timeout`` session variable to voluntarily request the server to automatically cancel any query using the current connection that runs over a specified length of time. For example, the following would cancel any query that runs for more 15 seconds automatically:
+
+.. code:: bash
+
+    => SET statement_timeout = 15000
+
+
+You may check the `Client Connection Defaults <https://www.postgresql.org/docs/9.6/runtime-config-client.html>`_ documentation for more information on the available session variables.
+
+.. note:: 
+
+    There may be a scenario where you encounter an error when running the above commands.
+
+If the database user is not a member of the database connected to, you may encounter the error:
```

Comment:
I believe this needs a separate section

Reply:
Created a new section called "Common errors".  Let me know if this is a good division or if it is better at a separate article entirely